src/styles/block/pruduct-card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  background-color: $white-color;
+  border: 1px solid #f3f3f3;
+  padding: 0 16px;
+
+  &__image {
+    justify-content: center;
+    display: flex;
+    height: 134px;
+    margin-top: 32px;
+  }
+
+  &__image-content {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__info {
+    margin-top: 40px;
+    margin-bottom: 0;
+    padding: 0;
+    color: $primary-color;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    text-align: left;
+  }
+
+  &__code {
+    display: flex;
+    margin-top: 4px;
+    height: 14px;
+    align-items: center;
+    color: $secondary-color;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: left;
+    font-weight: inherit;
```

Comment:
Using `font-weight: inherit` here might not be necessary unless you have a specific reason for this element to inherit its font-weight from its parent. If the font-weight should be the same as the default for the card, it's better to remove this property.